keyboards/crkbd/keymaps/soundmonster/rules.mk

```
@@ -1,5 +1,5 @@
 RGBLIGHT_ENABLE    = no
-RGB_MATRIX_ENABLE  = WS2812
+RGB_MATRIX_ENABLE  = yes
```

I guess the driver should be specified as well?
```suggestion
RGB_MATRIX_ENABLE  = yes
RGB_MATRIX_DRIVER   = WS2812
```

---

Nope, it's specified in the keyboard-level rules.mk now (that's the whole point of this PR).